core-services/prow/02_config/openshift/frr/_pluginconfig.yaml

```diff
@@ -0,0 +1,25 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/frr
+  require_self_approval: false
+label:
+  restricted_labels:
+    openshift/frr:
+    - allowed_users:
+      - russellb
+      - fedepaol
+      label: backport-risk-assessed
+    - allowed_users:
+      - asood-rh
+      - gregkopels
+      - evgenLevin
+      label: cherry-pick-approved
+lgtm:
+- repos:
+  - openshift/frr
+  review_acts_as_lgtm: true
+plugins:
+  openshift/frr:
+    plugins:
+    - approve
```

Comment:
approve plugin only makes sense with automated merges. see other components' `_prowconfig.yaml` file - it is probably easiest to adapt config from a component whose behavior you want to mimic. because you are setting up restrictions for `backport-risk-assessed` / `cherry-pick-approved`, I guess you are a part of the OCP process, so it would be best to adapt config from an OCP component